wiki/Glossary/en.md

```diff
@@ -658,43 +660,47 @@ Rank has three meanings:
 
 ### Ranking
 
-Ranking, alternatively called _leaderboard_, is a list that sorts members of the osu!community according to certain criteria. This can also refer to one's position in such a list.
+Ranking, alternatively called *leaderboard*, is a list that sorts members of the osu! community according to certain criteria. This can also refer to one's position in such a list.
 
 osu! currently has five rankings:
 
 - [PP](/wiki/pp) rankings: the ranking most commonly referred to, based on the player's pp.
 - Chart rankings: based on ranked score for beatmaps in a chart.
 - Score rankings: based on the player's ranked score.
-- Country rankings: based on the collective scores of player's per country.
+- Country rankings: based on the collective scores of players per country.
 - [Kudosu](/wiki/kudosu) rankings: based on the user's kudosu! count.
 
 ### Recalc
 
-Recalc is short for _Recalculate slider lengths_. Recalc is a command under the `Timing` header of the editor screen.
+*Short for "Recalculate slider lengths".*
+
+Recalc is a command under the `Timing` header of the editor screen.
 
 Usage of this is suggested after a [BPM](/wiki/bpm) change is made, if the beatmap already has sliders. This command will automatically adjust the length of every slider in the beatmap to the new BPM. If used, it is important, especially if a large change in BPM has been made, to check every slider and adjust its length manually, where necessary.
 
 ### Resnap
 
-Resnap is a pair of commands under the `Timing` header of the [editor](/wiki/editor) screen. One of which applies to the entire [beatmap](/wiki/beatmap) and the other to the current timing section (if the beatmap has only one timing section, both commands are the same).
+A pair of commands under the `Timing` header of the [editor](/wiki/editor) screen. One of which applies to the entire [beatmap](/wiki/beatmap) and the other to the current timing section (if the beatmap has only one timing section, both commands are the same).
 
 Usage is suggested when there has been a change in offset and/or BPM, after objects have been placed. This command will automatically reposition all objects on the timeline so that they fall on the closest beat division your snap divisor allows. If used, it is important, especially if a large change in BPM and/or offset has been made, to check every object and adjust its position on the [timeline](/wiki/timeline) manually, where necessary.
```

Comment:
```suggestion
Usage is suggested when there has been a change in offset and/or BPM, after objects have been placed. This command will automatically reposition all objects on the timeline so that they fall on the closest beat division your snap divisor allows. If used, it is important, especially if a large change in BPM and/or offset has been made, to check every object and adjust its position on the [timeline](/wiki/Beatmap_Editor/Timelines) manually, where necessary.
```